Userland/Libraries/LibColorScheme/ColorScheme.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Optional.h>
+#include <AK/Vector.h>
+#include <LibGfx/Color.h>
+
+namespace ColorScheme {
+
+Vector<String> get_color_scheme_names();
+
+class ColorScheme {
+public:
+    ErrorOr<void> set_color_scheme_from_string(StringView name);
+    Vector<Gfx::Color> get_colors() { return m_colors; };
+    Vector<Gfx::Color> get_bright_colors() { return m_bright_colors; };
+    Optional<Gfx::Color> get_background_color() { return m_background_color; };
+    Optional<Gfx::Color> get_foreground_color() { return m_foreground_color; };
```

Comment:
These APIs share a lot of properties with LibGfx themes. Is there a reason we shouldn't extend themes to apply to this use case?